definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -3905,13 +3905,41 @@
     "UserRole": "caseworker-divorce-financialremedy-courtadmin",
     "CRUD": "CRU"
   },
+  {
```

Comment:
should probs be behind a feature flag otherwise you won't be able to merge until it is released.